build.gradle

```diff
@@ -23,6 +23,50 @@ buildscript {
 
 plugins {
     id("io.gitlab.arturbosch.detekt").version("1.0.0-RC16")
+    id("com.nimbledroid.profiler").version("1.2.1")
```

Comment:
There's an option to add a custom label to the uploads: I wonder if this is useful or if it'd just spam our "My Apps" screen.